html/rendering/non-replaced-elements/the-fieldset-element-0/legend-margin-block.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<title>legend and margin (block-flow direction)</title>
+<link rel=match href=legend-margin-block-ref.html>
+<style>
+ body, fieldset { margin: 0; }
+ p { margin: 0; height: 50px; }
+ fieldset { border: 20px solid green; background: lime; padding: 10px; }
+ #top { margin-top: 100px; }
+ #bottom { margin-bottom: 100px; }
+ #behind {
+   position: absolute;
+   z-index: -1;
+   background: red;
+   left: 0;
+   right: 0;
+   top: 150px;
+   height: 220px;
```

Comment:
Is this really behavior we want to standardize on? Looks like this test expects the legend's top margin to collapse with that of the fieldset, even though the fieldset has a border, AND both elements establish a BFC.

Reply:
FWIW, I'd personally be okay with ignoring margin altogether (or just ignoring margin-top) as well.

Reply:
I guess we can ignore margin in the block flow direction. @whatwg/fieldset any opinions?

Reply:
I don't find the LEGEND block margins useful at all. Since interoperability is so lousy in this regard anyway, it would be tempting to say that we just ignore them.

Reply:
OK.

Reply:
Bug where margin-bottom was implemented for WebKit
https://bugs.webkit.org/show_bug.cgi?id=35981

Reply:
Thanks. There's nothing in that bug report that points out the usefulness of supporting margin-bottom on legend, though, is there? Might as well use padding-top on the fieldset, right?

Reply:
Yeah, I agree.

Reply:
> Thanks. There's nothing in that bug report that points out the usefulness of supporting margin-bottom on legend, though, is there? Might as well use padding-top on the fieldset, right?

Seems like it -- though in Firefox, you can use margin-bottom values to move the legend outside of the fieldset's border entirely (which you can't do with padding-top, but you could achieve with relative positioning instead).  That special margin-bottom behavior doesn't seem to happen in other browsers, so I'm happy to see that behavior go, anyway.

Reply:
One could use relative positioning to achieve that if desired.